### PR TITLE
chore: merge N+1 under guides

### LINKED ISSUE #256: merge n+1 pages
Merge all the N + 1 pages into one and move it under guide.

### PR DESCRIPTION
**Pull request by 35C4n0r**
/claim #256
Closes #256

### CODE REVIEW

**Comment by 35C4n0r** [comment]
@tusharmath do we also want to remove the previous N+1 section ?

**Comment by 35C4n0r** [comment]
@tusharmath PR is ready for review.

File: docs/n+1/canvas.excalidraw [hunk 1]
```diff
@@ -1,4088 +0,0 @@
-{
-  "type": "excalidraw",
```
Comment (tusharmath) [inline docs/n+1/canvas.excalidraw]: should keep this file.

**Review by tusharmath — CHANGES_REQUESTED** [review]
keep the canvas.

**Comment by 35C4n0r** [comment]
> keep the canvas.

@tusharmath done!

**Comment by 35C4n0r** [comment]
@tusharmath resolved the conflict.

**Review by tusharmath — APPROVED** [review]

**Pull request merged**